### PR TITLE
az-apim: add page

### PR DESCRIPTION
**Pull request by crixodia**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/az-apim.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# az apim
+
+> Manage Azure API Management services.
+> Part of `az`, the command-line client for Microsoft Azure.
```
Comment (kbdharun) [inline pages/common/az-apim.md:4]:
```suggestion
> Part of `az`.
```
Ref. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#descriptions

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by kbdharun — APPROVED** [review]

**Comment by blueskyson** [comment]
@kbdharun All Azure pages should remove "the command-line client for Microsoft Azure."
Screenshot from 2023-10-11 10-47-10

**Comment by kbdharun** [comment]
Agreed, will work on a PR for it later today.

**Pull request merged**